src/app/components/elements/modals/RequiredAccountInformationModal.tsx

```diff
@@ -72,7 +69,7 @@ const RequiredAccountInfoBody = () => {
             </div>
             {!isTutorOrAbove(user) && <div className="text-left mb-4">
                 Account type: <b>{user?.loggedIn && user.role && UserFacingRole[user.role]}</b> <span>
-                    <small>(Are you a teacher or tutor? {" "}
```

Comment:
Is this bracket meant to be paired with the one a couple lines down, after the link?

Reply:
Ah yes good spot; that was my VS Code unhappy with the bracket on its own, caused by {".)"} on line 78. Fixed it now!